tests/test_atime_ctime.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import division, print_function, unicode_literals
+
+from os import stat
+from copy import copy
+
+from libarchive import memory_reader, memory_writer, file_writer, \
+                       file_reader, extract_file
```

Comment:
Can you please use `(...)` for multiline imports. It would be also helpful to apply alphanumeric sort to imports.